src/tests/ftest/util/agent_utils.py

```diff
@@ -144,59 +110,59 @@ def run_agent(basepath, server_list, client_list=None):
                 output = sessions[client].stderr.read()
             except IOError as excpn:
                 if excpn.errno != errno.EAGAIN:
-                    raise AgentFailed("Error in starting daos_agent: "
-                                      "{0}".format(str(excpn)))
+                    raise AgentFailed(
+                        "Error in starting daos_agent: {0}".format(str(excpn)))
                 time.sleep(1)
                 continue
             expected_data += output
 
             match = re.findall(pattern, output)
             if len(match) > 0:
-                print("<AGENT> agent started on node {} in {} "
-                      "seconds".format(client, time.time() - start_time))
+                print(
+                    "<AGENT> agent started on node {} in {} seconds".format(
+                        client, time.time() - start_time))
                 break
 
     return sessions
-# pylint: enable=too-many-locals
 
 
 def stop_agent(sessions, client_list=None):
-    """
-    This should kill ssh and the agent.  This is temporary; presuming the
-    agent will deamonize at somepoint and can be started killed more
-    appropriately.
+    """Kill ssh and the agent.
 
-    client_list -- kill daos_agent on these hosts
-    sessions    -- set of subprocess sessions returned by run_agent()
-    """
+    This is temporary; presuming the agent will deamonize at somepoint and can
+    be started killed more appropriately.
```

Comment:
Typo/grammar here, only worth fixing if you have to push again for any other more significant reason.
```suggestion
    be started and killed more appropriately.
```

Reply:
Fixed - Thanks.